Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4456,6 +4456,12 @@ RefPtr<CSSStyleValue> Parser::parse_filter_value_list_value(TokenStream<Componen
                     return {};
                 if (amount->is_number() && amount->number().value() < 0)
                     return {};
+                if (first_is_one_of(filter_token, FilterToken::Grayscale, FilterToken::Invert, FilterToken::Opacity, FilterToken::Sepia)) {
+                    if (amount->is_percentage() && amount->percentage().value() > 100)
+                        amount = Percentage { 100 };
+                    if (amount->is_number() && amount->number().value() > 1)
+                        amount = Number { Number::Type::Integer, 1.0 };
+                }
```

Comment:
I suspect we should be doing this clamping at computed-value time instead of parse-time... But it's still better to have it than not have it.

Reply:
I'm not sure if here is the correct place to do this, but I'm not sure the unclamped value is observable at all in other browsers.

For reference [the spec](https://drafts.fxtf.org/filter-effects/#funcdef-filter-grayscale) says this: 
> Values of amount over 100% are allowed but UAs must clamp the values to 1

And the following example prints the clamped value:
```html
<!DOCTYPE html>
<style>
    div {
        width: 100px;
        height: 100px;
        background-color: green;
    }
</style>
<div></div>
<script>
    const div = document.querySelector("div");
    div.style.filter = "grayscale(200%)";
    console.log(div.style.filter); // prints "grayscale(100%)"
</script>
```